HeavyFlavorAnalysis/Onia2MuMu/interface/Onia2MuMuPAT.h

```diff
@@ -0,0 +1,83 @@
+#ifndef HeavyFlavorAnalysis_Onia2MuMu_Onia2MuMuPAT_h
+#define HeavyFlavorAnalysis_Onia2MuMu_Onia2MuMuPAT_h
+
+
+// system include files
+#include <memory>
+
+// FW include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDProducer.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+#include "CommonTools/Utils/interface/PtComparator.h"
+
+// DataFormat includes
+#include <DataFormats/PatCandidates/interface/CompositeCandidate.h>
+#include <DataFormats/PatCandidates/interface/Muon.h>
+
+#include <CommonTools/UtilAlgos/interface/StringCutObjectSelector.h>
+#include "RecoVertex/VertexTools/interface/InvariantMassFromVertex.h"
+
+#include "DataFormats/VertexReco/interface/VertexFwd.h"
+
+template<typename T>
+struct GreaterByVProb {
+  typedef T first_argument_type;
+  typedef T second_argument_type;
```

Comment:
@alberto-sanchez - I guess that these typedefs are not needed? They seem not to be used.